tsk/fs/fs_attr.c

```diff
@@ -275,7 +275,7 @@ tsk_fs_attr_set_str(TSK_FS_FILE * a_fs_file, TSK_FS_ATTR * a_fs_attr,
 uint8_t
 tsk_fs_attr_set_run(TSK_FS_FILE * a_fs_file, TSK_FS_ATTR * a_fs_attr,
     TSK_FS_ATTR_RUN * a_data_run_new, const char *name,
-    TSK_FS_ATTR_TYPE_ENUM type, uint16_t id, TSK_OFF_T size,
+    TSK_FS_ATTR_TYPE_ENUM type, int16_t id, TSK_OFF_T size,
```

Comment:
Is that per artifact, or total number of attributes for the case?

On Thu, May 12, 2016 at 2:41 PM, eugene7646 <notifications@github.com>
wrote:

> In tsk/fs/fs_attr.c
> <https://github.com/sleuthkit/sleuthkit/pull/649#discussion_r63074353>:
>
> > @@ -275,7 +275,7 @@ tsk_fs_attr_set_str(TSK_FS_FILE * a_fs_file, TSK_FS_ATTR * a_fs_attr,
> >  uint8_t
> >  tsk_fs_attr_set_run(TSK_FS_FILE * a_fs_file, TSK_FS_ATTR * a_fs_attr,
> >      TSK_FS_ATTR_RUN * a_data_run_new, const char *name,
> > -    TSK_FS_ATTR_TYPE_ENUM type, uint16_t id, TSK_OFF_T size,
> > +    TSK_FS_ATTR_TYPE_ENUM type, int16_t id, TSK_OFF_T size,
>
> Actually, according to Brian: "I don't think I've ever seen anything over
> 10".
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sleuthkit/sleuthkit/pull/649/files/c1af7e0a7d6a8f31199c761256d272ccd95dc9a6#r63074353>
>